docs/installation.rst

```diff
@@ -49,7 +49,7 @@ Cloning ACE
 As the ace user you previously created, cd into /opt and git clone the IntegralDefense ACE master branch: https://github.com/IntegralDefense/ACE.git::
 
   $ cd /opt
-  $ git clone https://github.com/IntegralDefense/ACE.git
+  $ git clone https://github.com/IntegralDefense/ACE.git && mv ACE ace
```

Comment:
you can do this instead
`git clone https://github.com/IntegralDefense/ACE.git ace`

Also don't forget that most of the time /opt is owned by root and not able to be modified by a non-root user.

Reply:
I took your suggestion, and in this case the ace user should already own the /opt dir if the install directions are being followed correctly